src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -15,7 +15,7 @@ module GEOS_LandAssimGridCompMod
   use ESMF
   use MAPL_Mod
   use ESMF_CFIOMOD,              only: ESMF_CFIOstrTemplate
-  
+  use GEOS_OutputIncrGridCompMod,only: OutputIncrSetServices=>SetServices  
```

Comment:
Re. the name of the new GridComp:
1) I think it would be better to use "Export" instead of "Output".  The term "export" suggests that this is about MAPL.  The term "output" is used in the legacy LDASsa subroutines that bypass MAPL.
2) I'm not sure if we should include "Land" or "Ldas" in the GC name.  This GC is in a sub-dir of the LandAssim GC, so it may not be necessary.
So maybe change "GEOS_OutputIncrGridCompMod" to "GEOS_ExportIncrGridCompMod" (with similar changes for the corresponding F90 file and dir name)

Reply:
I changed it to GEOS_ExportCatchIncrGridCompMod. It seems more clear now